website/versioned_docs/version-29.0/JestObjectAPI.md

```diff
@@ -837,44 +837,56 @@ This function is not available when using legacy fake timers implementation.
 
 ## Misc
 
-### `jest.setTimeout(timeout)`
```

Comment:
Sorry about messed up diffs here and in few following files. There are no changes in `jest.setTimeout()`. It was simply moved under `jest.retryTimes()`. Split view helps to make this readable at least.